v2/apiserver/config.go

```diff
@@ -140,11 +140,10 @@ func sessionsServiceConfig(
 		return config, err
 	}
 	if config.RootUserEnabled {
-		rootUserSessionTTL, err := os.GetRequiredEnvVar("ROOT_USER_SESSION_TTL")
-		if err != nil {
-			return config, err
-		}
-		config.RootUserSessionTTL, err = time.ParseDuration(rootUserSessionTTL)
+		config.RootUserSessionTTL, err = os.GetDurationFromEnvVar(
+			"ROOT_USER_SESSION_TTL",
+			time.Hour,
+		)
 		if err != nil {
 			return config, err
 		}
```

Comment:
Non-blocking nit: This error check can be collapsed into the previous line. I try to do that wherever I can get away with it. Again... non-blocking.

Reply:
Oh, definitely.  Missed that detail in your original recommendation.